packages/vaadin-crud/src/vaadin-crud-edit-column.js

```diff
@@ -63,17 +53,30 @@ class CrudEditColumnElement extends GridColumnElement {
     };
   }
 
-  /** @private */
-  ready() {
-    super.ready();
-    this.renderer = (root) => {
-      if (!root.firstElementChild) {
-        const elm = document.createElement('vaadin-crud-edit');
-        this.hasAttribute('theme') && elm.setAttribute('theme', this.getAttribute('theme'));
-        this.editLabel && elm.setAttribute('aria-label', this.ariaLabel);
-        root.appendChild(elm);
+  static get observers() {
+    return ['_onBodyTemplateOrRendererOrBindingChanged(_bodyTemplate, _renderer, _cells, _cells.*, path, ariaLabel)'];
+  }
+
+  /**
+   * Renders the crud edit element to the body cell.
+   *
+   * @override
+   */
+  _defaultRenderer(root, _column) {
+    let edit = root.firstElementChild;
+    if (!edit) {
+      edit = document.createElement('vaadin-crud-edit');
+      if (this.hasAttribute('theme')) {
+        edit.setAttribute('theme', this.getAttribute('theme'));
       }
-    };
+      root.appendChild(edit);
+    }
+
+    if (this.ariaLabel) {
```

Comment:
The [previous logic](https://github.com/vaadin/web-components/pull/2092/files#diff-7eae1c3c15528c069bd153562cd7e4f0bb70b6f4cba5c14839003ba21873caccL73 ) seems to check for `this.editLabel` instead?

Reply:
There was something weird. The `editLabel` check was always `false` in the renderer as we don't have `this.editLabel` defined in `vaadin-crud-edit-column`. 

And, in fact, `vaadin-crud` used to set the `aria-label` attribute [here](https://github.com/vaadin/web-components/pull/2092/files#diff-b44c16ded48c906cf79c5fd6f3e20ac052a8a44c41753502c9057c4ff50084cbL540-L542).

I've moved setting `aria-label` to the renderer as it was apparently planned before.